react/week1/todo/src/todoapp.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+//import ReactDOM from "react-dom";
```

Comment:
Remove that line. 
Avoid commented code if it is not important to the reviewer.